.bcr/metadata.template.json

```diff
@@ -0,0 +1,15 @@
+{
+    "homepage": "https://github.com/bazelbuild/rules_android_ndk",
+    "maintainers": [
+        {
```

Comment:
Should probably backfill some names here

Reply:
made an edit